src/localization/locales/en-US.json

```
@@ -844,6 +844,11 @@
         "home_draft_label": "Draft",
         "home_edit": "Edit",
         "home_default_description": "Use maps, photos, videos, audio recordings and narrative to create an impactful story.",
+        "delete_label": "Delete",
+        "delete_confirm_title": "Delete {{title}} story map?",
+        "delete_confirm_message": "You cannot undo deleting your story map. Deleting your published story map may cause a missing page error to your story map viewers.",
```

```suggestion
        "delete_confirm_message": "Are you sure you wish to delete the story map? This cannot be undone.,
```
Should match other deletion text.